docs/testing.md

```diff
@@ -92,7 +92,8 @@ is set to `true`.
 Quarks StatefulSet requires a k8s webhook to mutate the volumes of a pod.
 Kubernetes will call back to the operator for certain requests and use the
 modified pod manifest, which is returned.
-CF-Operator also uses a validating webhook to check the BOSH deployment custom resource.
+CF-Operator also uses a validating webhook to check the BOSH deployment custom resource and
```

Comment:
I'm wondering what is checked for those two. Could be more specific.

Reply:
Done